web/templates/filter_topics.hbs

```diff
@@ -1,6 +1,11 @@
-<div class="input-append topic_search_section filter-topics">
-    <input class="topic-list-filter home-page-input filter_text_input" id="filter-topic-input" type="text" autocomplete="off" placeholder="{{t 'Filter topics'}}" />
-    <button type="button" class="btn clear_search_button" id="clear_search_topic_button">
-        <i class="fa fa-remove" aria-hidden="true"></i>
-    </button>
+<div class="filter-topics">
+    <div class="input-append topic_search_section">
+        <input class="topic-list-filter home-page-input filter_text_input" id="filter-topic-input" type="text" autocomplete="off" placeholder="{{t 'Filter topics'}}" />
+        <button type="button" class="btn clear_search_button" id="clear_search_topic_button">
+            <i class="fa fa-remove" aria-hidden="true"></i>
+        </button>
+    </div>
+    {{#if stream_has_resolved_topics}}
+    <div class="resolved-toggle new-style"></div>
```

Comment:
Let's remove `new-style` class from here.

Reply:
@amanagr , so I am not sure why but I realized there is a class name called `tab-switcher` that the component->toggle adds to the div element which essentially has all the css content inside it, for all the other overlays it is working however for the resolved toggler, the CSS is not working, so basically if we remove `new-style` we have to copy-paste and adjust the tab-switcher css into the `left_sidebar.css` to make it work, so do let me know what you think about it.

Reply:
OK, we can clear than up later then.